pkg/api/v1/types.go

```diff
@@ -2412,6 +2412,12 @@ type PodSpec struct {
 	// +patchMergeKey=IP
 	// +patchStrategy=merge
 	HostAliases []HostAlias `json:"hostMappings,omitempty" patchStrategy:"merge" patchMergeKey:"IP" protobuf:"bytes,23,rep,name=hostMappings"`
+	// If specified, indicates the pod's priority. "system" is a special keyword
+	// which indicates the highest priority. Any other name must be defined in
+	// Admission Controller config or pod will be rejected.
```

Comment:
Let me call it "Priority" admission controller for now. If we decided that it is not a good name, I will change it.
And, yes, I would like to do admission controller for 1.7 as well if we reach a consensus on how priority should be defined and work.